src/vivarium_census_prl_synth_pop/components/population.py

```diff
@@ -41,7 +41,7 @@ def setup(self, builder: Builder):
         self.config = builder.configuration.population
         self.seed = builder.configuration.randomness.random_seed
         self.randomness = builder.randomness.get_stream(
-            "household_sampling", for_initialization=True
+            "household_sampling", initializes_crn_attributes=True
```

Comment:
We don't have CRN in this simulation, so `initializes_crn_attributes` should be (the default) False.

Reply:
@rmudambi Interesting, changing this actually causes the `test_addresses_during_moves` integration test to fail.